Makefile

```
@@ -102,6 +95,13 @@ kind-load-images: \
 		$(TRIVY_OPERATOR_IMAGE) \
 		$(TRIVY_OPERATOR_IMAGE_UBI8)
 
+.PHONY: deploy
+deploy: manifests kind-load-images
+	kubectl apply --server-side -k deploy/static
```

```suggestion
	kubectl apply -k deploy/static
```

The PR suggests to use server-side apply, which is the new preferred way of applying resources to your cluster. And will become the new default in Kubernetes at some point. But I have no strong feelings for this, and can go back to a standard (client-side) apply.